packages/tools/cli/src/models/index.ts

```diff
@@ -1,11 +1,7 @@
-import Commands from './commands';
+export { default as Commands } from './commands';
```

Comment:
This is an interesting export. I've never seen this before. Is there a guide for this?

Reply:
Why doesn't `export Commands from './commands';` work?

Reply:
This is a common pattern used to optimize for **tree-shaking**. To improve compatibility with various bundlers and tooling, it is typically the most efficient to never re-declare a only-exported value at its modular de-mark point.

i.e:

```ts
// ./src/models/example/index.ts

import Example from './class.ts'; // this is okay here, as this is the lowest leaf in this scope that we attend to be within the public API

export type { ExampleType } from './types.ts'; // this is a standard type-only re-export. ESLint-TS-Plugin will throw if not done this way

export default Example;

```

```ts
// ./src/models/index.ts

export { default as Example } from './example'; // this is a single line export with no locally declared namespaces, thus tree-shaking is even less likely to miss-interpret this as an IIFE and not tree-shake

export type { ExampleType } from './example'; // standard type re-export, see previous comment above.
```